python/tk_multi_publish2/context_widget/ui/context_editor_widget.py

```diff
@@ -60,6 +60,7 @@ def setupUi(self, ContextWidget):
         self.task_menu_btn.setMaximumSize(QtCore.QSize(32, 32))
         self.task_menu_btn.setFocusPolicy(QtCore.Qt.NoFocus)
         self.task_menu_btn.setLayoutDirection(QtCore.Qt.LeftToRight)
+        self.task_menu_btn.setStyleSheet("QToolButton::menu-indicator { image: none; }")
```

Comment:
This should be done in this file: https://github.com/shotgunsoftware/tk-multi-publish2/blob/master/style.qss...

And looks like it already is: https://github.com/shotgunsoftware/tk-multi-publish2/blob/master/style.qss#L387

So, I'm wondering why MOBU doesn't respect it in the `.qss` file. If this works, then we can leave it here.